keel-tagging-plugin/src/test/kotlin/com/netflix/spinnaker/keel/tagging/KeelTagHandlerTests.kt

```diff
@@ -109,7 +109,8 @@ internal class KeelTagHandlerTests : JUnit5Minutests {
     "keel-tag",
     ResourceMetadata(
       name = ResourceName(keelId),
-      uid = randomUID()
+      uid = randomUID(),
+      serviceAccount = "keel@spinaker"
```

Comment:
I approve of the @HORSE_SPINAKER spelling

Reply:
😆 oops